packages/auth0-ai-langchain/auth0_ai_langchain/token_vault/__init__.py

```diff
@@ -0,0 +1,10 @@
+from auth0_ai.interrupts.token_vault_interrupt import (
+    TokenVaultError as TokenVaultError,
+    TokenVaultInterrupt as TokenVaultInterrupt
+)
+
+from auth0_ai.authorizers.token_vault_authorizer import (
+    get_credentials_from_token_vault as get_credentials_from_token_vault,
+    get_access_token_for_connection as get_access_token_for_connection
```

Comment:
`get_access_token_for_connection` should be removed from everywhere. Spotted this reference but there are more.

Reply:
9f43ca15843bf93c31c500e718849dca941c26d7